docs/components/01-preview.mdx

```diff
@@ -0,0 +1,28 @@
+# Preview
+
+<Description
+  importExample="import { Preview } from '@smile/react-front-kit';"
+  packageInfo={{
+    label: '@smile/react-front-kit',
+    link: 'https://www.npmjs.com/package/@smile/react-front-kit',
+  }}
+  sourceInfo={{
+    link: 'https://github.com/Smile-SA/react-front-kit/blob/main/packages/react-front-kit/src/Components/Preview/Preview.tsx',
+  }}
+>
+  Renders a docs, pdf, png, svg, video and audio documents preview
```

Comment:
Plus simplement "Renders a preview of different types (image, video, audio or an iframe for any other types) from a source URL"